guides/source/active_record_callbacks.md

```diff
@@ -608,23 +933,36 @@ class PictureFile < ApplicationRecord
 end
 ```
 
-NOTE: The `:on` option specifies when a callback will be fired. If you don't supply the `:on` option the callback will fire for every action.
+NOTE: The `:on` option specifies when a callback will be fired. If you don't
+supply the `:on` option the callback will fire for every action.
```

Comment:
Maybe instead of "action" we say "live cycle event"?

I was also wondering if we should connect this back to "Registering Callbacks to Fire on Lifecycle Events", where we introduce the `:on` option.